.github/workflows/ci.yml

```diff
@@ -80,6 +80,13 @@ jobs:
           echo "$(pwd)" >> $GITHUB_PATH
           ./bundletool version
 
+      - name: Setup cwebp
+        run: |
+          curl -L https://storage.googleapis.com/downloads.webmproject.org/releases/webp/libwebp-1.5.0-linux-x86-64.tar.gz -o webp.tar.gz
+          tar -xzf webp.tar.gz
+          echo "$(pwd)/libwebp-1.5.0-linux-x86-64/bin" >> $GITHUB_PATH
+          ./libwebp-1.5.0-linux-x86-64/bin/cwebp -version
```

Comment:
weirdly the $GITHUB_PATH seems to only apply to the next step and not to this one so we have to call it directly.
seems like the same pattern was used for bundletool

Reply:
Ah i was trying to answer the question you asked before you asked it but I didn’t explain it well. Maybe my comment below helps with the link to the example? https://github.com/getsentry/launchpad/pull/65/files#r2166018284